go.mod

```diff
@@ -7,13 +7,13 @@ replace k8s.io/client-go => k8s.io/client-go v0.20.0
 require (
 	github.com/evanphx/json-patch v4.9.0+incompatible
 	github.com/go-logr/logr v0.3.0 // indirect
-	github.com/rancher/eks-operator v1.0.6
+	github.com/rancher/eks-operator v1.0.9
 	github.com/rancher/fleet/pkg/apis v0.0.0-20210225010648-40ee92df4aea
-	github.com/rancher/lasso v0.0.0-20200905045615-7fcb07d6a20b
-	github.com/rancher/norman v0.0.0-20210225010917-c7fd1e24145b
-	github.com/rancher/rancher/pkg/apis v0.0.0-20210409044207-9b0058c0329f
-	github.com/rancher/rancher/pkg/client v0.0.0-20210409044207-9b0058c0329f
-	github.com/rancher/rke v1.2.8-rc1.0.20210409005002-9213a4d08d08
+	github.com/rancher/lasso v0.0.0-20220506212308-e3ef9b77cc49
+	github.com/rancher/norman v0.0.0-20220712163932-620fef760449
+	github.com/rancher/rancher/pkg/apis v0.0.0-20220811200500-ace7b2c56632
+	github.com/rancher/rancher/pkg/client v0.0.0-20220811200500-ace7b2c56632
+	github.com/rancher/rke v1.2.22-rc1
```

Comment:
Do we maybe want to wait for this to a non-rc value before we put this in? Should be fine, just thought it was worth checking

Reply:
Looks like it was already set to an earlier rc before? This was auto-updated, not my change. I don't think we can wait on this, we can update again once the final one is cut.

Reply:
Agree, more about remembering to change this later. Will be a nesting-doll operation of rc's.